### ISSUE TITLE
Improve deployment

### ISSUE THREAD

**Issue by mbtamuli** [rtCamp/github-actions-library#21]
Deployments can feel slow because we are using [deployer](https://deployer.org). Deployer creates releases on each deploy and copies all the files. This is required in case of production environments to have an easy way to rollback and ensure all files are from a fresh WordPress install and none of them have been edited or corrupted.

But, for non-production environments, we can significantly improve the speed of deployment by simply copying the files using rsync(which will only copy newer files).

tl;dr - 

Production(master) - `deployer`
Non-production - `rsync`

**Comment by sudhir-yadav** [comment]
We can improve deployment speed, here are a few changes made to deployer script.
https://github.com/rtCamp/apogee/blob/master/.circleci/deploy.php

**Comment by mbtamuli** [comment]
Just found out `cachetool` has been turned into a recipe - https://deployer.org/recipes/cachetool.html. Can directly use this.
